Sample Applications/WPFGallery/ViewModels/MainWindowViewModel.cs

```diff
@@ -12,6 +12,39 @@ public partial class MainWindowViewModel : ObservableObject
     [ObservableProperty]
     private string _applicationTitle = "WPF Gallery Preview";
 
+    private ObservableCollection<string> SearchItems { get; set; }
```

Comment:
What's the need for search items ?

Reply:
It contains the list of all pages in string format. This is then used to filter out from this whole set without changing the original FilteredSearchItems.